exercises/calculator.cpp

```diff
@@ -10,3 +10,42 @@
   Multiplication: 8
   Division: 2
 */
+
+
+#include<iostream>
+
+
+using namespace std;
+
+
+int main() {
+  
+  int a,b;
+
+  //Inserting first number
+  cout << "Insert first number: ";
+  cin >> a;
+  cout << endl;
+
+  //Inserting second number
+  cout << "Insert second number: ";
+  cin >> b;
+  cout << endl;
+
+  cout << "Insert first number: " << a << "\n Insert second number: " << b << endl << endl;
+
+  cout << "sum: " << a+b << endl;
+  cout << "difference: " << a-b << endl;
+  cout << "multiplication: " << a*b << endl;
+
+  if(b!=0)
+    cout << "division: " << a/b << endl;
+  else {
```

Comment:
ottimo 👍 , potresti però migliorare la leggibilità aggiungendo le parentesi e mettendo qualche spazio in più
```suggestion
  if (b != 0) {
    cout << "division: " << a/b << endl;
  } else {
```